circle.yml

```diff
@@ -1,7 +1,15 @@
 machine:
   node:
     version: 7.8
+  # Specific yarn version needed until 0.28 is stable
+  # see https://github.com/devtools-html/perf.html/issues/439
+  post:
+    - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 0.28.4
+  environment:
+    PATH: "$HOME/.yarn/bin:$PATH"
 dependencies:
+  cache_directories:
+    - "~/.yarn" # speedup yarn installation
```

Comment:
note: from your PR, I looked into `~/.cache/yarn`, but I think this is mostly useful on a desktop, when you can run yarn install on several new packages; because we cache `node_modules` already I think this is not useful to cache `~/.cache/yarn` as well.